.github/workflows/build-push-image.yml

```diff
@@ -0,0 +1,31 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.main'
+      - 'pearson-release/olive.stage'
```

Comment:
I think this branch is not going to be used @Squirrel18?

Reply:
@alexjmpb  I noticed that Andrés left the same comment in your PR, so I'll remove it.